### ISSUE TITLE
Music "Repeat" behaviour inconsistent

### ISSUE THREAD

**Issue by BSmith-66** [jellyfin/jellyfin-androidtv#729]
First Bug here - be gentle, pretty sure this doesn't fit the media issue category

**Describe the bug**
Repeat functionality inconsistent
when playing music, more often than not the same track will be repeated rather than progress to the next track in the queue even though repeat is off. 

**To Reproduce**
1. Load up album or playlist or instant mix ensure that repeat is off
2. Play song to completion
3. high chance track repeats 
4. loose a few points in WAF

**Expected behaviour**
next track in the queue commences playback 

**Logs**
I'll need to figure out how to get them from android-TV i think i have the reporting opted in. 
multiple playlists, artists, users, media types mp3, flac, same hardware though, different servers
has persisted across versions 

**Screenshots**
N/A

**System (please complete the following information):**
 - Mibox3 -persisted versions, but currently on android 9 Pie - consistent this client
 - Android TV version - persisted multiple versions, currently on the dec-20 release (latest play store as of writing)
 - Jellyfin server version:= persisted multiple versions currently 10.6.4 stable, haven't tried 10.7 RC's 
 - used multiple server fresh installs (other reasons), proxmox LXC deb or Ubuntu base, docker on PI4

**Additional context**
tried letting screen saver auto kick in to see if it had effect
moving to menu but music playing in background
keeping the playlist foreground
toggling repeat on and off - noticed that with repeat 1 enabled sometimes it would go to next track
basically if it goes to the next track or repeats is 50:50
may happen at 1st track or 4th or 10th but it will happen
can manually skip to next track and wait in anticipation again. 
- I have a feeling the repeat function is kinda broken.  

appreciate all the effort and the fact that every release gets better and better. 
If I knew how to fix it I would help, but I can help by raising a issue and testing any potential fixes.
